include/cutlass/gemm/dispatch_policy.hpp

```diff
@@ -269,6 +270,21 @@ struct MainloopSm90ArrayTmaGmmaWarpSpecialized {
     "KernelSchedule must be one of the Ptr-Array or Grouped Gemm TMA Warp Specialized Cooperative policies");
 };
 
+
+#if defined(CUTLASS_ENABLE_SYCL)
+struct MainloopIntelPVCBase {
+  constexpr static int Stages = 1;
+  using ArchTag = arch::IntelPVC;
+  using Schedule = KernelSinglestage;
+  using ClusterShape = Shape<_1,_1,_1>;
+  static constexpr int SG_SZ = 16;
```

Comment:
Could you use CamelCase and the full name to match the naming convention above?

```suggestion
static constexpr int SubgroupSize = 16;
```